firstTimeInstall.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+locale  # check for UTF-8
+
+sudo apt update && sudo apt install locales
+sudo locale-gen en_US en_US.UTF-8
+sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+export LANG=en_US.UTF-8
+
+locale  # verify settings
+
+sudo apt install software-properties-common
+sudo add-apt-repository universe
+
+sudo apt update && sudo apt install curl -y
+sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
+
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+sudo apt update
+
+sudo apt upgrade
+
+sudo apt install \
+ros-humble-desktop \
+ros-humble-ros-base \
+ros-dev-tools \
+python3-dev \
+python3-pip \
+clang-format \
+clang-tidy
+
+source /opt/ros/humble/setup.bash
+
+echo source /opt/ros/humble/setup.bash >> ~/.bashrc
```

Comment:
if members run this more than once this is going to get called and make .bashrc look like shit

Maybe modify bash script to include something like this

`if ! grep -q "source /opt/ros/humble/setup.bash" ~/.bashrc; then
  echo "source /opt/ros/humble/setup.bash" >> ~/.bashrc
  echo "ROS 2 sourced in bashrc"
fi`

Reply:
Nice addition